### ISSUE TITLE
some diffs

### ISSUE THREAD

**Issue by banayoyo** [tadax/m2det#5]
for code about SFAM as bellowing, seems fc1 is sigmoid and fc2 is relu, while the order in author' code is inverse?
 
            with tf.variable_scope('SFAM'):
                for i in range(self.scales):
                    feature = tf.concat([outs[j][i] for j in range(self.levels)], axis=3)
                    attention = tf.reduce_mean(feature, axis=[1, 2], keepdims=True)
                    attention = tf.layers.dense(inputs=feature, units=64, 
                                                activation=tf.nn.sigmoid, name='fc1_{}'.format(i+1))
                    attention = tf.layers.dense(inputs=attention, units=1024,
                                                activation=tf.nn.relu, name='fc2_{}'.format(i+1))
                    feature = feature * attention
                    features.insert(0, feature)

**Comment by tadax** [comment]
@banayoyo Thank you for your comment. It was a great help. I believe eq. (2) is wrong in M2Det paper.

> s = F_ex(z, W) = σ(W_2 · δ(W_1 · z)), (2)
> where σ refers to the ReLU function, δ refers to the sigmoid function

Activation function for last layer should be sigmoid.
